frontend/packages/pipelines-plugin/src/plugin.tsx

```diff
@@ -29,7 +29,7 @@ import {
 import * as models from './models';
 import * as pipelineIcon from './images/pipeline.svg';
 import { pipelinesTopologyPlugin } from './topology/pipelinesTopologyPlugin';
-import { TopologyDataModelFactory } from '@console/dev-console/src/extensions/topology';
+import { TopologyDataModelFactory } from '@console/topology/src/extensions/topology';
```

Comment:
```suggestion
import { TopologyDataModelFactory } from '@console/topology';
```
Another one you could search for and replace as well.